LSP.sublime-settings

```diff
@@ -49,7 +49,7 @@
   },
 
   // Gutter marker for code diagnostics.
-  // Valid values are "bookmark", "circle", "cross", "dot" or ""
+  // Valid values are "bookmark", "circle", "cross", "dot", "sign" or ""
```

Comment:
BTW. `dot` doesn't work.
It's not part one of the standard icons. Documentation says:
```
An optional unicode string specifying an icon to draw in the gutter next to each region. The icon will be tinted using the color associated with the scope. Standard icon names are "dot", "circle" and "bookmark". The icon may also be a full package-relative path, such as "Packages/Theme - Default/dot.png".
```
We should probably remove that option.

Reply:
`dot` does work for me. You can even use just `"branch"` from the default theme in LSP, even though it is not documented.

Reply:
Sorry, I meant "cross" doesn't work.

Reply:
Can confirm and removed it. By the way, `label` and `warning` (from the default theme) have the correct icon size and work as well, even though undocumented.